07-multiple-regression.Rmd

```diff
@@ -434,18 +434,8 @@ Let's revisit the instructor evaluation data introduced in Section \@ref(model1)
 
 Let's reload the `evals` data and `select()` only the needed subset of variables. 
 
-```{r eval=FALSE}
-load(url("http://www.openintro.org/stat/data/evals.RData"))
-evals <- evals %>%
-  select(score, age, gender)
-```
-
-```{r echo=FALSE}
-if(!file.exists("data/evals.RData")){
-  download.file(url = "http://www.openintro.org/stat/data/evals.RData", 
-                destfile = "data/evals.RData")
-}
-load(file = "data/evals.RData")
+```{r eval=TRUE}
+data("evals")
```

Comment:
@rudeboybert Do we still want to do non-lazy load here? What are the benefits instead of just printing `evals`? I can't remember.